covasim/webapp/index.html

```diff
@@ -102,64 +102,61 @@ <h5 class="modal-title" id="exampleModalLabel">About Covasim</h5>
       </div>
 
       <!-- Run button -->
-      <div class="btn-group btn-group-lg">
-        <button v-if="running" class="btn btn-primary green" type="button" disabled  style="width:160px">
+      <div class="d-flex align-items-center flex-wrap">
+        <button v-if="running" class="btn btn-primary btn-lg btn-run green" type="button" disabled>
           <span class="spinner-border spinner-border-sm" role="status" aria-hidden="true"></span>
           Running...
         </button>
-        <button v-else id="run_btn" class="btn btn-primary green" type="button" @click="runSim" style="width:160px">
+        <button v-else id="run_btn" class="btn btn-primary btn-lg btn-run green" type="button" @click="runSim">
           Run
-        </button>&nbsp;&nbsp;&nbsp;&nbsp;
-        <button id="reset_btn" :disabled="running" class="btn btn-primary blueish" type="button" @click="resetPars">
+        </button>
+        <button id="reset_btn" :disabled="running" class="btn btn-primary btn-lg ml-2 mr-2 blueish" type="button" @click="resetPars">
           Reset
-        </button>&nbsp;&nbsp;&nbsp;&nbsp;
-        <b-tooltip target="reset_btn" triggers="hover">
+        </button>
+        <b-tooltip target="reset_btn" triggers="hover focus">
```

Comment:
Maybe we can add a tooltip partial, there are a bunch scattered around and it maybe nice to have a simple function that can build one?

Reply:
I guess they call them components 😅 https://vuejs.org/v2/guide/#Composing-with-Components